code/mob/living.dm

```diff
@@ -1870,6 +1870,12 @@ var/global/icon/human_static_base_idiocy_bullshit_crap = icon('icons/mob/human.d
 	var/rangedprot_base = get_ranged_protection() //will be 1 unless overridden
 	if (P.proj_data) //Wire: Fix for: Cannot read null.damage_type
 		var/rangedprot_mod = max(rangedprot_base*(1-P.proj_data.armor_ignored),1)
+
+		if (rangedprot_mod > 1)
+			armor_msg = ", but your armor softens the hit!"
+		else
+			armor_msg = ", but [P] pierces through your armor!"
```

Comment:
should only print the message if you actually *had* armor to be pierced - i.e.: `if (rangedprot_base > 1)`